src/cadence/testnet/getNFTs.cdc

```diff
@@ -281,12 +281,16 @@ pub fun getGaia(owner: PublicAccount, id: UInt64): NFTData? {
 }
 
 pub fun stringStartsWith(string: String, prefix: String): Bool {
+    if(string.length < prefix.length) {
+        return false
+    }
+
     let beginning = string.slice(from: 0, upTo: prefix.length)
 
     let prefixArray = prefix.utf8
     let beginningArray = beginning.utf8
 
-    for index in prefixArray {
+    for index, element in prefixArray {
```

Comment:
This element variable was causing the script to fail, is this part of a new flow-cli version?

Reply:
It's been in the flow docs for at least 6 months: https://github.com/onflow/cadence/commit/1f37d4144ed7ffc4410e4e621fd3aeac55591de3

The previous version will definitely break though as it's indexing by element instead of by index